<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakePacket</class>
 <widget class="QDialog" name="MakePacket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QComboBox*/

QComboBox {
border: 1px solid rgb(111, 156, 207);
border-radius: 3px;
padding: 1px 18px 1px 3px;
min-width: 9em;
max-width:9em;
}
QComboBox::drop-down {
subcontrol-origin: padding;
subcontrol-position: top right;
width: 15px;
border-left-width: 1px;
border-left-color: darkgray;
border-left-style: solid;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}
QComboBox QAbstractItemView {
border: 2px solid darkgray;
selection-background-color: lightgray;
}


/*QPushButton*/

QPushButton{
min-width:75px;
max-width:75px;
min-height:20px;
border:1px solid black;
border-radius:5px;
}
QPushButton:hover{
background-color: #333333;
border-color: #444444;
color:rgb(255,255,255);
}
QPushButton:pressed{
background-color: #111111;
border-color: #333333;
color: yellow;
}

/*QTabWidget*/
QTabWidget::tab-bar {
alignment: left;
}
QTabWidget::pane {
border-top: 2px solid #C2C7CB;
}
QTabBar::tab:selected {
margin-top:0px;
border-color: #9B9B9B;
border-bottom-color: #C2C7CB;
}
QTabBar::tab:!selected {
margin-top: 2px;
}
QTabBar::tab {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
border: 2px solid #C4C4C3;
border-bottom-color: #C2C7CB;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
min-width: 8ex;
padding: 2px;
}

/*QWidget*/
QWidget{
background-color:#F5F5F5;
}

/*QGroupBox*/
QGroupBox {
border: 2px solid gray;
border-radius: 5px;
margin-top: 1ex;
}
QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top center;
padding: 0 3px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}

/*QCheckBox*/

QCheckBox::indicator{
width: 13px;
height: 13px;
background-color: #00FF00;
border: 1px solid #000000;
border-radius:4px;
}
QCheckBox::indicator::checked{
background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15,
radius:0.3, stop:0 #FF0000, stop:1 #FFFFFF);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>控制主机信息报文</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="2">
                  <widget class="QCheckBox" name="checkBox">
                   <property name="text">
                    <string>自增</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_4">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_3">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>控制主机软件主版本：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>控制主机生命信号：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>控制主机软件次版本：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="lineEdit_5">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>控制主机命令报文</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>除尘:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>开/关门：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>本地/远程：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_3">
               <item>
                <property name="text">
                 <string>本地</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>远程</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>运行/维护：</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comboBox_4">
               <item>
                <property name="text">
                 <string>运行</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>维护</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>面阵开/关:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBox_5">
               <item>
                <property name="text">
                 <string>开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>线阵开/关：</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBox_6">
               <item>
                <property name="text">
                 <string>开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>加热开/关：</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QComboBox" name="comboBox_7">
               <item>
                <property name="text">
                 <string>开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>散热开/关：</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QComboBox" name="comboBox_8">
               <item>
                <property name="text">
                 <string>开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>模拟测试触发指令：</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QComboBox" name="comboBox_9">
               <item>
                <property name="text">
                 <string>测试开</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>测试关</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>生成报文</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>报文类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>发送周期：ms</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>数据：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
